deploy/helm/kubecf/assets/operations/instance_groups/brain-tests.yaml

```diff
@@ -37,6 +37,15 @@
           password: ((cf_admin_password))
           skip_ssl_validation: true
           user: admin
+        tests:
+          autoscaler:
+            enabled: {{ .Values.features.autoscaler.enabled }}
+          tcprouting:
+            enabled: {{ .Values.features.routing_api.enabled }}
+          insecure_registry:
```

Comment:
Technically this should be fixed in the test script instead, or have it fail fast when it hits an invalid configuration. But whatever, that's okay too.

Reply:
It's the responsibility of KubeCF to tell the test what should and what should not be executed.

Reply:
It's an implementation detail that the insecure registry test depends on TCP routing, though?